flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -731,7 +731,8 @@ function wait_for_restart_to_complete {
     local expected_num_restarts=$((current_num_restarts + 1))
 
     echo "Waiting for restart to happen"
-    while ! [[ ${current_num_restarts} -eq ${expected_num_restarts} ]]; do
+    while [[ ${current_num_restarts} -le ${expected_num_restarts} ]]; do
```

Comment:
For the record, this should have been `-lt` and it was fixed before pushing to master.